frontend/src/app/AdminNavBar/AdminNavBar.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import { BiCommentDetail } from 'react-icons/bi'
+import { Link as ReactLink } from 'react-router-dom'
+import {
+  As,
+  chakra,
+  Flex,
+  FlexProps,
+  HStack,
+  useDisclosure,
+} from '@chakra-ui/react'
+
+import { BxsHelpCircle } from '~assets/icons/BxsHelpCircle'
+import { ReactComponent as BrandMarkSvg } from '~assets/svgs/brand/brand-mark-colour.svg'
+import { useIsMobile } from '~hooks/useIsMobile'
+import { logout } from '~services/AuthService'
+import IconButton from '~components/IconButton'
+import Link from '~components/Link'
+import { AvatarMenu, AvatarMenuDivider } from '~templates/AvatarMenu/AvatarMenu'
+
+import { EmergencyContactModal } from '~features/user/emergency-contact/EmergencyContactModal'
+import { useUser } from '~features/user/queries'
+
+import Menu from '../../components/Menu'
+
+const BrandSmallLogo = chakra(BrandMarkSvg)
+
+const NAV_LINKS = [
+  {
+    label: 'Feature request',
+    href: '',
+    MobileIcon: BiCommentDetail,
+  },
+  {
+    label: 'Help',
+    href: 'https://guide.form.gov.sg',
+    MobileIcon: BxsHelpCircle,
+  },
+]
+
+type AdminNavBarLinkProps = {
+  label: string
+  href: string
+  MobileIcon?: As
+}
+
```

Comment:
/superNit, since these are the same types, we could make it explicit:

```suggestion
type AdminNavBarLinkProps = {
  label: string
  href: string
  MobileIcon?: As
}

const NAV_LINKS: AdminNavBarLinkProps[] = [
  {
    label: 'Feature request',
    href: '',
    MobileIcon: BiCommentDetail,
  },
  {
    label: 'Help',
    href: 'https://guide.form.gov.sg',
    MobileIcon: BxsHelpCircle,
  },
]
```

Also, do we have the url for `Feature request`?

Reply:
added, do we have a file for links?